### DISCUSSION TITLE
update from Prisma 3 to Prisma 4

### DISCUSSION THREAD

**Discussion by lhocke** [blitz-js/blitz#3468] [Ideas] [answered]
Prisma 4 has implemented a number of QOL changes including defaults on scalar lists, explicit 1-1 unique constraints, and a fair number of updates to queries https://www.prisma.io/docs/guides/upgrade-guides/upgrading-versions/upgrading-to-prisma-4

**Comment by beerose** [comment, marked as answer]
Hey! We will definitely upgrade to Prisma 4 — I was already looking into it and there are no breaking changes. If you have an existing Blitz project you can upgrade it yourself too.

**Reply by lhocke** [reply]
awesome, thought it might be in the works already but figured I'd put it on the discussion board anyway.
